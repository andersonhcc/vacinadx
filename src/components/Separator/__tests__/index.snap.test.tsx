/**
 * @format
 */

import 'react-native';
import React from 'react';
import {Separator} from '../';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('Separator', () => {
  it('should render without props', () => {
    const {toJSON} = renderer.create(<Separator />);

    expect(toJSON()).toMatchSnapshot();
  });

  it('should render with height prop', () => {
    const {toJSON} = renderer.create(<Separator height={15} />);

    expect(toJSON()).toMatchSnapshot();
  });

  it('should render with width prop', () => {
    const {toJSON} = renderer.create(<Separator width={15} />);

    expect(toJSON()).toMatchSnapshot();
  });

  it('should render with width and height props', () => {
    const {toJSON} = renderer.create(<Separator width={15} height={15} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
