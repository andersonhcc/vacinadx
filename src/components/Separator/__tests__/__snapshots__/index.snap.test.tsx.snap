// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator snap should render with height prop 1`] = `
<View
  height={15}
  style={
    [
      {
        "height": 15,
        "maxHeight": 15,
        "minHeight": 15,
      },
    ]
  }
  testID="separator"
/>
`;

exports[`Separator snap should render with width and height props 1`] = `
<View
  height={15}
  style={
    [
      {
        "height": 15,
        "maxHeight": 15,
        "maxWidth": 15,
        "minHeight": 15,
        "minWidth": 15,
        "width": 15,
      },
    ]
  }
  testID="separator"
  width={15}
/>
`;

exports[`Separator snap should render with width prop 1`] = `
<View
  style={
    [
      {
        "maxWidth": 15,
        "minWidth": 15,
        "width": 15,
      },
    ]
  }
  testID="separator"
  width={15}
/>
`;

exports[`Separator snap should render without props 1`] = `
<View
  style={
    [
      {},
    ]
  }
  testID="separator"
/>
`;
